operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1167,6 +1199,21 @@ querier:
   env: []
   extraEnvFrom: []
 
+  kedaAutoscaling:
+    enabled: false
+    minReplicaCount: 2
+    maxReplicaCount: 10
+    pollingInterval: 10
+    querySchedulerInflightRequestsThreshold: 6
```

Comment:
The jsonnet calculates with as 0.75*8 where 8 is the default value for max_concurrent requests. In Helm that's 16. Let's use 12 here in this case